apps/test/unit/javalab/JavalabViewTest.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {expect} from '../../util/reconfiguredChai';
+import {mount} from 'enzyme';
+import JavalabView from '@cdo/apps/javalab/JavalabView';
+import {Provider} from 'react-redux';
+import pageConstants, {setPageConstants} from '@cdo/apps/redux/pageConstants';
+import {
+  getStore,
+  registerReducers,
+  stubRedux,
+  restoreRedux
+} from '@cdo/apps/redux';
+import javalab, {toggleDarkMode} from '@cdo/apps/javalab/javalabRedux';
+import color from '@cdo/apps/util/color';
+global.$ = require('jquery');
```

Comment:
Curious: Why is this needed? I would have expected jquery to already exist here.

Reply:
JavalabView gave me an error without it so I guess it didn't exist